src/app/view/components/login/login-component/login-container.component.ts

```diff
@@ -30,8 +31,10 @@ export class LoginContainerComponent {
         private dotRouterService: DotRouterService,
         private httprequestUtils: HttpRequestUtils,
         private loggerService: LoggerService,
-        private loginService: LoginService
+        private loginService: LoginService,
+        private dotLoadingIndicatorService: DotLoadingIndicatorService
     ) {
+        this.dotLoadingIndicatorService.hide();
```

Comment:
Why do you need it right here? by default the loading indicator it hidden, right? if not I think we have to make it hidden by default.